docs/reference/tests/_index.mdx

```diff
@@ -0,0 +1,3 @@
+This section goes over different techniques for testing Nitric applications locally, and in the cloud.
+
+We attempt to make the local run of nitric as similar as possible as the cloud environments, so these tests should act the same locally as they do the cloud. However, running them on the cloud may incur costs.
```

Comment:
"as similar as possible as the cloud environments" to "as similar as possible to the cloud environments"?